shark/shark_importer.py

```diff
@@ -122,3 +122,15 @@ def forward(self, inputs=None):
             print("Inference", self.model_source_hub, " not implemented yet")
         elif self.model_source_hub == "jaxhub":
             print("Inference", self.model_source_hub, " not implemented yet")
+
+
+def shark_load(model_name, file_path):
+    file_link = f"https://storage.googleapis.com/shark_tank/users/stanley/{model_name}.mlir"
```

Comment:
Can't this be generic file_link, instead of specifying user_name etc.

Reply:
I suggested users/ for custom user generated models. The official models once they are loaded into shark_tank/<model_name> should be simple like HF namespace.